nvm.sh

```diff
@@ -1444,7 +1540,35 @@ nvm_print_versions() {
   local LTS
   local FORMAT
   local NVM_CURRENT
+  local NVM_LATEST_LTS_COLOR
+  local NVM_OLD_LTS_COLOR
+
+  local NVM_CURRENT_COLOR
+  local NVM_INSTALLED_COLOR
+  local NVM_SYSTEM_COLOR
+  local NVM_NOT_INSTALLED_COLOR
+  local NVM_DEFAULT_COLOR
+  local NVM_LTS_COLOR
+
+  if [ -n "${NVM_COLORS-}" ]; then
+    NVM_CURRENT_COLOR=$(nvm_get_colors 1)
+    NVM_INSTALLED_COLOR=$(nvm_get_colors 2)
+    NVM_SYSTEM_COLOR=$(nvm_get_colors 3)
+    NVM_NOT_INSTALLED_COLOR=$(nvm_get_colors 4)
+    NVM_DEFAULT_COLOR=$(nvm_get_colors 5)
+    NVM_LTS_COLOR=$(nvm_get_colors 6)
+  else
+    NVM_CURRENT_COLOR='0;32m'
+    NVM_INSTALLED_COLOR='0;34m'
+    NVM_SYSTEM_COLOR='0;33m'
+    NVM_NOT_INSTALLED_COLOR='0;31m'
+    NVM_DEFAULT_COLOR='0;37m'
+    NVM_LTS_COLOR='1;33m'
+  fi
```

Comment:
that way, this could be:
```suggestion
  NVM_CURRENT_COLOR=$(nvm_get_colors 1)
  NVM_INSTALLED_COLOR=$(nvm_get_colors 2)
  NVM_SYSTEM_COLOR=$(nvm_get_colors 3)
  NVM_NOT_INSTALLED_COLOR=$(nvm_get_colors 4)
  NVM_DEFAULT_COLOR=$(nvm_get_colors 5)
  NVM_LTS_COLOR=$(nvm_get_colors 6)
```